examples/zo/scripts/memory.py

```
@@ -0,0 +1,245 @@
+    )
+    if result.returncode != 0:
+        if allow_doctor_partial and args_list and args_list[0] == "doctor":
+            text = result.stdout or result.stderr or ""
+            try:
+                data = json.loads(text)
+                checks = data.get("checks", [])
+                critical_failures = [c for c in checks if not c.get("ok") and c.get("check") != "Config file"]
+                if not critical_failures:
+                    return text
+            except Exception:
+                pass
```

Ruff (0.15.12)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)